src/containers/About/about.jsx

```diff
@@ -3,30 +3,27 @@ import propTypes from 'prop-types';
 import { ReactComponent as Illustration } from '../assets/images/about.svg';
 import { ReactComponent as Logo } from '../assets/images/logo.svg';
 import Aboutdata from './data';
-import MainHeading from '../../components/MainHeading/index';
-import BackButton from '../../components/BackButton/index';
-
+import Header from '../../components/Header';
 import './about.css';
 
 const About = props => {
   const {
     history: { goBack },
   } = props;
   return (
-    <div className="about">
-      <div className="about__heading">
-        <BackButton handleBack={goBack} />
-        <MainHeading text="About" />
+    <>
+      <Header text="About" handleBack={goBack} />
+      <div className="about">
+        <Illustration />
+        {Aboutdata.map(data => (
```

Comment:
aboutdata should be in camel case 
```suggestion
        {aboutData.map(data => (
```